diff --git a/node_modules/@fast-csv/format/build/src/formatter/RowFormatter.js b/node_modules/@fast-csv/format/build/src/formatter/RowFormatter.js
index 25555db..e67be33 100644
--- a/node_modules/@fast-csv/format/build/src/formatter/RowFormatter.js
+++ b/node_modules/@fast-csv/format/build/src/formatter/RowFormatter.js
@@ -5,7 +5,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.RowFormatter = void 0;
 const lodash_isfunction_1 = __importDefault(require("lodash.isfunction"));
-const lodash_isequal_1 = __importDefault(require("lodash.isequal"));
+const { isDeepStrictEqual } = require('node:util');
 const FieldFormatter_1 = require("./FieldFormatter");
 const types_1 = require("../types");
 class RowFormatter {
@@ -128,7 +128,7 @@ class RowFormatter {
             return { shouldFormatColumns: true, headers: null };
         }
         // if the row is equal to headers dont format
-        return { shouldFormatColumns: !(0, lodash_isequal_1.default)(headers, row), headers };
+        return { shouldFormatColumns: !isDeepStrictEqual(headers, row), headers };
     }
     // todo change this method to unknown[]
     gatherColumns(row) {
